src/test/util/transaction_utils.cpp

```diff
@@ -36,3 +38,33 @@ CMutableTransaction BuildSpendingTransaction(const CScript& scriptSig, const CTr
 
     return txSpend;
 }
+
+std::vector<CMutableTransaction> SetupDummyInputs(FillableSigningProvider& keystoreRet, CCoinsViewCache& coinsRet, const std::array<CAmount,4>& nValues)
```

Comment:
I see added new function `SetupDummyInputs` but don't see removed old copies in:
 - src/bench/ccoins_caching.cpp
 - src/test/transaction_tests.cpp